lib/axlsx_styler.rb

```diff
@@ -8,3 +8,24 @@
 Axlsx::Workbook.send :include, AxlsxStyler::Axlsx::Workbook
 Axlsx::Worksheet.send :include, AxlsxStyler::Axlsx::Worksheet
 Axlsx::Cell.send :include, AxlsxStyler::Axlsx::Cell
+
+module Axlsx
```

Comment:
I'd keep this part here, because it's the only place we override / complement `axlsx`'s methods.